config/packages/security.yaml

```diff
@@ -1,11 +1,4 @@
 security:
-    enable_authenticator_manager: true
-
-    encoders:
```

Comment:
die encoders gaan nu automatisch?

Reply:
Die config optie bestaat niet meer. Denk dat volgende lijn uit de UPGRADE.md relevant is:

> Drop all classes in the Core\Encoder\ sub-namespace, use the PasswordHasher component instead